hackthebox/user.py

```diff
@@ -142,6 +142,7 @@ def __init__(self, data: dict, client: "HTBClient", summary: bool = False):
         self.name = data["name"]
         self.user_owns = data["user_owns"]
         self.points = data["points"]
+        self.avatar = f'https://www.hackthebox.com{data["avatar"]}'
```

Comment:
The `avatar` property is currently documented as being the relative URL of the avatar (e.g. `/storage/...`) (even if it wasn't actually implemented before). The documentation for the property should either be updated to reflect this.

Additionally, I'd like to avoid having hardcoded URLs like this as it makes testing harder. Instead, I'd prefer the `constants` file to have:

```py
URL_BASE "https://www.hackthebox.com"
API_BASE = f"{URL_BASE}/api/v4/"
```
and this line updated to use this property.